docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md

```diff
@@ -1563,6 +1563,380 @@ properly by the operating system.
 
 1. Pressing the button once should increase the keyboard backlight.
 
+## ECR025.001 Permanent keyboard illumination after cold-boot (Firmware)
```

Comment:
As we add new test cases it will be good to actualize test matrixes

Reply:
Test matrices for Novaustom platforms are already including new tests. It looks as below:

```bash
| 4.   | [Embedded Controller and Super I/O initialization][ECR]| ECR           | Without ECR010.001 and ECR010.002    |
```

Reply:
NV4x_TGL : `| Without ECR021.xxx - ECR024.xxx      |`
NV4x_ADL: `| Without ECR021.xxx - ECR024.xxx      |`